src/main/resources/portal.properties.EXAMPLE

```diff
@@ -202,3 +202,10 @@ ucsc.build=hg19
 
 # default view in oncoprint (sample, patient (default))
 oncoprint.defaultview=patient
+
+# only set the custom annotation menu labels if we want custom annotations to appear 
+custom_annotation.binary.menu_label =
+custom_annotation.multi.menu_label =
```

Comment:
Is it important to have a property to name the labels in the OncoPrint? This applies a _cbioportal specific property_ to what might be a _study specific property_. I would suggest just calling them in the Oncoprint:
- Custom driver/passenger
- Custom driver class